### DISCUSSION TITLE
Cruiser Desktop - How can I move waypoints?

### DISCUSSION THREAD

**Discussion by GuG1050** [devemux86/cruiser#394] [Q&A] [answered]
I wanted to use Cruiser desktop to edit and fine tune a Kurviger route. I manage to load a Kurviger file and its overlay however I fail to simpliy move existing or newly added shaping or via points. From Basecamp and Kurviger-Web I am used to drag and drop. But how can I do this in Cruiser desktop (Windows)?

**Comment by devemux86** [comment, marked as answer]
On desktop use **Space** or **Caps Lock** + **mouse drag**  to move waypoints, favorites.
This extra step is there to avoid accidentally moving points when moving the map.

You can find all desktop keyboard shortcuts here:
- https://wiki.openstreetmap.org/wiki/Cruiser#Keyboard_shortcuts

**Reply by devemux86** [reply]
It should probably be optional via a setting.

**Reply by GuG1050** [reply]
Thank you. It is ok as it is. You only need to know 🙂.
